src/app/shared/utils/utils.spec.ts

```diff
@@ -3,7 +3,7 @@
 import { Utils } from './utils';
 import { NgbDateStruct } from '@ng-bootstrap/ng-bootstrap';
 
-describe('sdfasdfas', () => {
+describe('Test convertFormGroupNGBDateToJSDate and convertJSDateToNGBDate methods.', () => {
```

Comment:
haha, i like the other name better... just kidding.  Looks like someones cat named it.